packages/mediacenter/kodi-theme-Estuary/package.mk

```diff
@@ -50,7 +50,6 @@ makeinstall_target() {
     cp -R */ $INSTALL/usr/share/kodi/addons/skin.estuary
     cp *.txt $INSTALL/usr/share/kodi/addons/skin.estuary
     cp *.xml $INSTALL/usr/share/kodi/addons/skin.estuary
-    cp *.png $INSTALL/usr/share/kodi/addons/skin.estuary
```

Comment:
No, `icon.png` has moved to the resources folder after https://github.com/xbmc/xbmc/pull/10185

The `resources` folder is already copied in line 50.